launch/LMS1xx.launch

```diff
@@ -1,6 +1,13 @@
 <launch>
   <arg name="host" default="192.168.1.14" />
+  
+  <!-- start and end units are in degrees to keep with lms internal structure. They will be converted to rad in the node. -->
+  <arg name="start" default="-999.9" />
+  <arg name="end" default="999.9" />
```

Comment:
Just looking at merging this now, sorry for the delay. Can you clarify why these defaults are +/-999? Isn't the maximal sweep of the device ~4.71 rad (270 degrees)?